src/server/debug.rs

```diff
@@ -746,9 +795,169 @@ impl<ER: RaftEngine> DebuggerImpl<ER> {
             None => Err(Error::NotFound(format!("region {}", region_id))),
         }
     }
+
+    fn exec_flashback(
+        &self,
+        mut need_flashback_regions: Vec<u64>,
+        version: u64,
+        start_ts: TimeStamp,
+        commit_ts: TimeStamp,
+        f: impl Fn(&Self, u64, Vec<u8>, Vec<u8>, Context, bool, TimeStamp, TimeStamp) -> Result<()>,
+    ) -> Result<()> {
+        let timeout = Duration::from_secs(FLASHBACK_TIMEOUT);
+        let start_time = Instant::now();
+
+        let mut ctx = Context::default();
+        let store_id = self.get_store_ident().unwrap().get_store_id();
+        loop {
```

Comment:
It seems that this loop will flashback the regions one by one, is it possible to flashback them concurrently?

Reply:
Good idea! updated